### DISCUSSION TITLE
Domoticz and Homeassistant

### DISCUSSION THREAD

**Discussion by platte-73** [Koenkk/zigbee2mqtt#15233] [General]
Can I use one zigbee2mqtt installation for domoticz and Homeassistant

I now have Synolgy with docker domoticz and docker zigbee2mqtt. I want to install a Homeassistant docker and also use the same zigbee2mqtt docker installation. Is this possible to have both runing and use the same zigbee2mqtt

**Comment by SVH-Powel** [comment]
Yes, I would think that should be OK

**Reply by platte-73** [reply]
Now running Domoticz and Home assistant on one zigbee2mqtt instance
